src/generation.rs

```diff
@@ -3511,4 +3541,142 @@ fn add_struct_derives<T: DataType>(data_type: &mut T, used_in_key: bool, is_enum
             }
         }
     }
+}
+
+fn generate_int(gen_scope: &mut GenerationScope, types: &IntermediateTypes) {
+    let ident = RustIdent::new(CDDLIdent::new("int"));
+    // todo: wasm bindings + to/from string traits
```

Comment:
todo? If this is meant to be done in a separate PR, let's create an issue for it and then link the issue from the comment

Reply:
Oops. I forgot to remove this comment. They're implemented in this PR.